lib/routes/evenings.js

```diff
@@ -31,9 +33,11 @@ module.exports = router
             .then(evening => res.json(evening))
             .catch(next);
     })
-    
+
     .get('/', ensureAuth, (req, res, next) => {
         const { query } = req;
+        // Do we trust this query enough to pass it
+        // directly to our database?
         Evening.find({ ...query, user: req.user._id })
```

Comment:
Don't trust user input. Parse the query before you pass it to your database. Make sure you know what you are sending your db.